core-test/src/test/scala/org/bitcoins/core/protocol/ln/ShortChannelIdTest.scala

```diff
@@ -36,7 +36,7 @@ class ShortChannelIdTest extends BitcoinSUnitTest {
       .fromHumanReadableString("0x0x65536")
     an[IllegalArgumentException] must be thrownBy ShortChannelId
       .fromHumanReadableString("0x0x-1")
-    an[NoSuchElementException] must be thrownBy ShortChannelId
+    an[IllegalArgumentException] must be thrownBy ShortChannelId
```

Comment:
Is indicative on an unintended change?

Reply:
This is caused from the change in `CryptoBytesUtil.decodeHex()` which now uses `ByteVector.fromValidHex()`. A `ShortChannelId` is not valid hex, which is why a `IllegalArgumentException` gets thrown.